src/utils/NucleicAcidSearchEngine.cpp

```diff
@@ -189,6 +189,9 @@ class NucleicAcidSearchEngine :
     setValidFormats_("exp_ms2_out", ListUtils::create<String>("mzML"));
 
     registerFlag_("decharge_ms2", "Decharge the MS2 spectra for scoring", true);
+    
+    registerFlag_("deisotope_ms2", "Deisotope the MS2 spectra for scoring", true);
```

Comment:
If you don’t make it a flag but a string parameter with restriction true false it will be easier to change the default. Just something to consider…